documentation/experimental_features.md

```diff
@@ -219,13 +219,19 @@ Futures will continue to execute in parallel. However, Bolt itself will not exit
 have completed in order to ensure that all work is finished. Any errors raised after the calling
 plan has finished will be logged at warn level for visibility.
 
-The `wait()` plan function accepts a single Future object or an array of Futures and blocks until
-they finish, with an optional timeout. If provided a timeout, any unfinished Futures will raise a
-timeout error if they have not completed within the timeout. You can return errors
-instead of raising them by passing `_catch_errors => true` to `wait()`. The `wait()`
-function returns the results from the Future blocks once they've all finished. If a
-Future errors, Bolt only raises the error after all other Futures finish executing and
-return to `wait()`.
+The `wait()` plan function accepts either a single Future object, an array of Futures, or no
+Futures. If not provided any Futures, the function implicitly waits for all Futures created up
+to that point in the calling plan. The `wait()` function does not wait on any Futures created
+in subplans or inside of `background()` blocks, even if the subplan is called inside a 
+`background()` block. If you want a plan to wait on a Future object, you must call `wait()` 
+in the subplan or `background()` block directly, or return Future objects to the calling plan 
+to be explicitly passed to `wait()`.
```

Comment:
I think an example describing how to properly wait on futures from sub-plans would be helpful.

Reply:
Ended up including a different (simpler) example, especially since I think now that blocks are less complicated (since we're not using the scope to determine wait ID) the futures from subplans bit is less confusing.